### PR TITLE
chore: fix urls

### PR DESCRIPTION
**Pull request by leangseu-edx**

### CODE REVIEW

File: src/data/redux/app/selectors.js [hunk 1]
```diff
@@ -55,7 +58,7 @@ export const courseCard = StrictDict({
     isRestricted: certificate.isRestricted,
   })),
   course: mkCardSelector(({ course }) => ({
-    bannerImgSrc: process.env.LMS_BASE_URL + course.bannerImgSrc,
+    bannerImaSrc: baseAppUrl(course.bannerImgSrc),
```
Comment (nsprenkle) [inline src/data/redux/app/selectors.js:61]: Typo
```suggestion
    bannerImgSrc: baseAppUrl(course.bannerImgSrc),
```

File: .env.test [hunk 2]
```diff
@@ -36,3 +36,4 @@ ENTERPRISE_MARKETING_URL='http://example.com'
 ENTERPRISE_MARKETING_UTM_SOURCE='example.com'
 ENTERPRISE_MARKETING_UTM_CAMPAIGN='example.com Referral'
 ENTERPRISE_MARKETING_FOOTER_UTM_MEDIUM='Footer'
+LEARNING_MICROFRONTEND_URL='http://localhost:2000'
```
Comment (nsprenkle) [inline .env.test:39]: Add missing newline
```suggestion
LEARNING_MICROFRONTEND_URL='http://localhost:2000'

```

File: .env.development [hunk 3]
```diff
@@ -36,3 +36,4 @@ ENTERPRISE_MARKETING_URL='http://example.com'
 ENTERPRISE_MARKETING_UTM_SOURCE='example.com'
 ENTERPRISE_MARKETING_UTM_CAMPAIGN='example.com Referral'
 ENTERPRISE_MARKETING_FOOTER_UTM_MEDIUM='Footer'
+LEARNING_MICROFRONTEND_URL='http://localhost:2000'
```
Comment (nsprenkle) [inline .env.development:39]: Add missing newline
```suggestion
LEARNING_MICROFRONTEND_URL='http://localhost:2000'

```

File: .env.test (hunk 2 above)
Comment (nsprenkle) [reply .env.test:39]: See other 2 instances as well

File: src/containers/CourseCard/components/CourseCardActions/ViewCourseButton.jsx [hunk 4]
```diff
@@ -17,7 +17,7 @@ export const ViewCourseButton = ({ cardId }) => {
     <Button
       disabled={isMasquerading || !hasAccess || (isEntitlement && isExpired)}
       as="a"
-      href={marketingUrl}
+      href={homeUrl}
```
Comment (muselesscreator) [inline src/containers/CourseCard/components/CourseCardActions/ViewCourseButton.jsx:20]: This should stay as marketingUrl, I believe

File: src/data/redux/app/selectors.js [hunk 5]
```diff
@@ -68,6 +71,12 @@ export const courseCard = StrictDict({
     isFinished: courseRun.isFinished,
     minPassingGrade: Math.floor(courseRun.minPassingGrade * 100),
     startDate: new Date(courseRun.startDate),
+    homeUrl: courseRun.homeUrl,
+    marketingUrl: learningMfeUrl(courseRun.marketingUrl),
```
Comment (muselesscreator) [inline src/data/redux/app/selectors.js:75]: Not sure about the others, but this one is not in the learning MFE.

Comment (muselesscreator) [reply src/data/redux/app/selectors.js:75]: (pretty sure...)

**Review by muselesscreator — CHANGES_REQUESTED** [review]
Couple of concerns around marketing URL, but beyond that looks good.

**Comment by muselesscreator** [comment]
also looks like you are going to have some conflicts w/ my PR from end of week

**Review by muselesscreator — COMMENTED** [review]
LGTM as long as merges nicely :-p

**Review by muselesscreator — APPROVED** [review]
LGTM

**Pull request merged**